pkg/kp/rcstore/consul_store.go

```diff
@@ -10,13 +10,20 @@ import (
 	"github.com/square/p2/Godeps/_workspace/src/github.com/hashicorp/consul/api"
 	"github.com/square/p2/Godeps/_workspace/src/github.com/pborman/uuid"
 
+	"github.com/square/p2/pkg/kp"
 	"github.com/square/p2/pkg/labels"
 	"github.com/square/p2/pkg/logging"
 	"github.com/square/p2/pkg/pods"
 	"github.com/square/p2/pkg/rc/fields"
 )
 
-const kvPrefix = "replication_controllers"
+const (
+	RC_TREE string = "replication_controller"
```

Comment:
My dream would be to fragment kp into several packages by type (pods, health, sessions/locks, rcs, orchestrator-pairs) and the top level package would only contain the constants and the general-purpose consul client initialization code. However that change is probably too big to pursue atm. For now I think a good policy is to only add new "tree types" as subpackages (as was done with rcstore) and slowly migrate other trees to subpackages as the opportunity arises (hopefully I can do that for locks). If we don't add any more trees to the main package then eventually it'll shrink to something more manageable.

As for whether RC_TREE should be in rcstore or kp/constants.go, I'm pretty indifferent. Originally I put it here because rcstore didn't need to import kp, but now that it uses LockPath, that's no longer relevant. I'll probably move it back to constants.go since it makes sense for them to be in the same place.